tasks/gotest.py

```diff
@@ -50,7 +51,7 @@
 WINDOWS_MAX_PACKAGES_NUMBER = 150
 TRIGGER_ALL_TESTS_PATHS = ["tasks/gotest.py", "tasks/build_tags.py", ".gitlab/source_test/*"]
 OTEL_UPSTREAM_GO_MOD_PATH = (
-    "https://raw.githubusercontent.com/open-telemetry/opentelemetry-collector-contrib/main/go.mod"
+    f"https://raw.githubusercontent.com/open-telemetry/opentelemetry-collector-contrib/v{OCB_VERSION}/go.mod"
```

Comment:
OCB version can differ from upstream collector version. It happened for v0.108.0

Reply:
they're pretty tightly coupled now, it's not currently possible with the way the release script works in opentelemetry-collector-releases upstream to release an ocb version without a new collector version to match